src/components/UpdatePasswordForm/UpdatePasswordForm.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC, useState } from 'react';
+import { Formik, Form, FormikHelpers } from 'formik';
+import { LoadingButton as Button } from '@mui/lab';
+import { Box } from '@mui/material';
+import * as yup from 'yup';
+import Input from '../Input';
+import { profile } from '../../services';
+import { useNotification } from '../../providers/NotificationProvider';
+import Translate from '../Translate';
+
+interface IFormValue {
+  oldPassword: string;
+  password: string;
+  confirmPassword: string;
+}
+
+const validationSchema = yup.object().shape({
+  oldPassword: yup.string().required(),
+  password: yup.string().min(8).max(50).required(),
+  confirmPassword: yup
+    .string()
+    .oneOf([yup.ref('password')], 'Password must match')
+    .required(),
+});
+
+const useUpdatePasswordForm = () => {
+  const [loading, setLoading] = useState(false);
+  const { showNotification } = useNotification();
+  const handleSubmit = (
+    value: IFormValue,
+    { setFieldError, resetForm }: FormikHelpers<IFormValue>
+  ) => {
```

Comment:
What do you think, maybe create a hook with the name for example `useFormikSubmit` or similar, I don't know

Reply:
good idea. but most cases has more differences than similarities.